app/assets/stylesheets/application.css.scss

```diff
@@ -13,8 +13,205 @@
+  margin-bottom: 20px;
+  //Instead of the line below you could use @include box-shadow($shadow-1, $shadow-2, $shadow-3, $shadow-4, $shadow-5, $shadow-6, $shadow-7, $shadow-8, $shadow-9, $shadow-10)
+  box-shadow: 0 10px 10px $black_20;
+  tr {
+    border-top: 1px solid $color_green_white_approx;
+  }
+  tbody {
+    background: $color_celeste_approx;
+  }
+  thead tr {
+    border-bottom: 1px solid $color_green_white_approx;
+    border-top: 1px solid $color_tradewind_approx;
+    background: $color_tradewind_approx;
+    color: $white;
+    &:first-child {
+      border-top: none;
+    }
+  }
+}
+
+tr {
+  &:nth-child(even) td {
+    background: linear-gradient(to bottom, $color_merino_approx 0, $color_ecru_white_approx 100%);
+  }
+  &:nth-child(odd) td {
+    background: $color_romance_approx;
+  }
+}
+
+.table>thead>tr>th {
+  @extend %extend_1;
+  vertical-align: bottom;
+  border-bottom: 2px solid $color_alto_approx;
+  text-transform: uppercase;
+}
+
+.table>thead>tr>td {
+  @extend %extend_1;
+}
+
+.table>tbody>tr>th {
+  @extend %extend_1;
+}
+
+.table>tbody>tr>td {
+  @extend %extend_1;
+}
+
+.table>tfoot>tr>th {
+  @extend %extend_1;
+}
+
+.table>tfoot>tr>td {
+  @extend %extend_1;
+}
+
+.btn-primary {
+  color: $white;
+  background-color: $color_fuscous_gray_approx;
+  border-color: $color_celeste_approx;
+  //Instead of the line below you could use @include border-radius($radius, $vertical-radius)
+  border-radius: 0;
+  &:hover {
+    color: $white;
+    background-color: $black;
+    border-color: $black;
+  }
+}
+
+.navbar-brand {
+  float: left;
+  padding: 15px;
+  font-size: 28px;
+  line-height: 20px;
+  height: 50px;
+  font-weight: bold;
+  text-transform: uppercase;
+  font-family: $font_5, $font_0, $font_3, $font_2, $font_4;
+  span {
+    font-weight: normal;
+  }
+  &:hover {
+    text-decoration: none;
+  }
+  &:focus {
+    text-decoration: none;
+  }
+}
+
+.navbar-default {
+  background-color: $color_fuscous_gray_approx;
+  border-color: $color_celeste_approx;
+  //Instead of the line below you could use @include box-shadow($shadow-1, $shadow-2, $shadow-3, $shadow-4, $shadow-5, $shadow-6, $shadow-7, $shadow-8, $shadow-9, $shadow-10)
+  box-shadow: 1px 1px 10px $black;
+  .navbar-brand {
+    color: $white;
+  }
+  .navbar-nav>li>a {
+    color: $white;
+    font-weight: bold;
+    &:hover {
+      color: $color_celeste_approx;
+      background-color: transparent;
+    }
+    &:focus {
+      color: $color_celeste_approx;
+      background-color: transparent;
+    }
+  }
+}
+
+p {
+  font-family: $font_0, $font_6, $font_7, $font_8, $font_4;
+}
+
+ol {
+  font-family: $font_0, $font_6, $font_7, $font_8, $font_4;
+}
+
+ul {
+  font-family: $font_0, $font_6, $font_7, $font_8, $font_4;
+}
+
+td {
+  font-family: $font_0, $font_6, $font_7, $font_8, $font_4;
+}
+
+a {
+  &:hover {
+    background-color: transparent;
+    outline: none;
+  }
+  &:active {
+    outline: none;
+  }
+}
+
+.row h1 {
+  font-weight: bold;
```

Comment:
Properties should be ordered background, color, font-size, font-weight, margin-bottom, padding